docs/user-guide/helm.md

```diff
@@ -47,6 +47,37 @@ spec:
 
 See [here](../operator-manual/declarative-setup.md#helm-chart-repositories) for more info about how to configure private Helm repositories.
 
+## Repository alias
+
+You can use the repository alias, as you have named it inside the repositoriesn as a repoURL in the source:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sealed-secrets
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: sealed-secrets
+    repoURL: @sealed-secrets # supports also the format alias:sealed-secrets
+    targetRevision: 1.16.1
+    helm:
+      releaseName: sealed-secrets
+  destination:
+    server: "https://kubernetes.default.svc"
+    namespace: kubeseal
+```
+
+Do not forget to add the corresponding entry in the project source repositories whitelist, if needed:
+
+```yaml
+spec:
+  sourceRepos:
+  - '@sealed-secrets'
```

Comment:
Should we add as well `sealed-secrets` ?

Reply:
No, just the format defined in the source.
So if you want both syntaxes you should have `@sealed-secrets` and `alias:sealed-secrets`

Reply:
Sorry yes I wanted to say `alias:sealed-secrets` could we add it in the comment